src/components/Dashboard/SortList/sortUtils.js

```diff
@@ -0,0 +1,31 @@
+export const sortComparer = (field, direction = 'ASC') => (a, b) => {
+  if (direction === 'NONE') {
+    return 0;
+  }
+
+  let compareA = String(a[field]).toLowerCase();
+  let compareB = String(b[field]).toLowerCase();
+
+  if (field === 'priority') {
+    compareA = String(['low', 'medium', 'high'].indexOf(compareA));
+    compareB = String(['low', 'medium', 'high'].indexOf(compareB));
+  }
+
+  const directionMultiplier = direction === 'ASC' ? 1 : -1;
+  return compareA.localeCompare(compareB) * directionMultiplier;
+};
+
+export const storage = {
+  get: (key) => {
+    let data = localStorage.getItem('SortList_Data') || '{}';
```

Comment:
This `data` isn't the best option too. I'm sure you can come up with a name that immediately identifies the variable for what it is

Reply:
sortListDataRaw = get from local storage
sortListData = parse local storage data